src/JuliusSweetland.OptiKey.UnitTests/UI/ViewModels/MainViewModelSpecifications/WhenConstruct.cs

```diff
@@ -547,8 +547,8 @@ protected override void Arrange()
         [Test]
         public void ThenKeyboardShouldBeSizeAndPosition()
         {
-            Assert.IsNotNull(MainViewModel.Keyboard);
-            Assert.AreEqual(typeof(OptiKey.UI.ViewModels.Keyboards.SizeAndPosition), MainViewModel.Keyboard.GetType());
+            Assert.That(MainViewModel.Keyboard, Is.Not.Null);
```

Comment:
This is one of the few cases where it's incorrect to use the Assert.Multiple method. We need MainViewModel.Keyboard to be NOT NULL for the following assert.